staging/src/k8s.io/apimachinery/pkg/util/strategicpatch/types.go

```diff
@@ -66,6 +66,9 @@ func (item *kindItem) VisitPrimitive(schema *openapi.Primitive) {
 	item.err = errors.New("expected kind, but got primitive")
 }
 
+func (item *kindItem) VisitArbitrary(schema *openapi.Arbitrary) {
```

Comment:
Don't know if this is the right fix, but it appears to have been a newer requirement by kube-openapi https://github.com/kubernetes/kube-openapi/issues/27